src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+package ip.src.main.java;
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
+    private static Task[] tasks = new Task[100];
+    private static String[] letter = new String[100];
+    private static int i = 0;
+
+    public static void addTask(Task t, int type) {
+        tasks[i] = t;
+        i++;
+        if (type == 1) letter[i] = "D";
+        else if (type == 2) letter[i] = "E";
+        else if (type == 3) letter[i] = "T";
+    }
+
+    public static void printTask(Task[] args) {
```

Comment:
Perhaps if the print and add functions could be set aside in the Task class for organisation